fabfile.py

```diff
@@ -64,7 +69,10 @@ def get_config():
         sys.exit(1)
 
     aws_config = AWSConfig(env.aws)
-    project_config = ProjectConfig(env.config, env.environment)
+    if env.passwords is not None:
+        project_config = ProjectConfig(env.config, env.environment, passwords=env.passwords)
+    else:
+        project_config = ProjectConfig(env.config, env.environment)
```

Comment:
This if block isn't needed since 'passwords' argument has a default value of None anyway, and the method checks for it.